src/theme/index.js

```diff
@@ -68,6 +69,12 @@ const theme = createTheme({
       primary: CHART_PRIMARY_COLOR_SCHEME,
       secondary: CHART_SECONDARY_COLOR_SCHEME,
     },
+    // To avoid issues with components that have default color
+    // see https://stackoverflow.com/a/72571943
+    default: {
+      main: grey[300],
+      dark: grey[400],
+    },
```

Comment:
How many places do we use color="default"? If not many, I think the best strategy is to change those places to use palette colors (primary, secondary, etc.).

Reply:
@kilemensi I have checked and this is not used anywhere. So let me remove it.

Reply:
Is it removed @kelvinkipruto ?

Reply:
I still see this here @kelvinkipruto

Reply:
@kilemensi We're not using it in our code, but removing it causes `build` to fail. I guess there is a component that uses it by default.
For context, I did not write the code myself, but it was added when running the [migration codemod](https://mui.com/material-ui/migration/migration-v4/#run-codemods) provided by MUI.

Let me take one more deep look into it and see if how we can resolve this.

Reply:
I know that @kelvinkipruto & hence why I'm asking for us to test and figure out which component is using it & update that component.

Reply:
@koechkevin

Reply:
@kelvinkipruto looks like you must have default object inside palette as is. 
Seems to be an issue with emotionjs.

Reply:
@kilemensi I couldn't find the component that uses this, but it still breaks if removed. I could use some help here cc @koechkevin